README.md

```diff
@@ -3,7 +3,20 @@
   <h3> <a href="https://taichi.readthedocs.io/en/latest/"> Documentation </a> | <a href="https://taichi.readthedocs.io/en/latest/hello.html"> Tutorial </a> | <a href="https://github.com/yuanming-hu/difftaichi"> DiffTaichi </a> | <a href="https://github.com/yuanming-hu/taichi/tree/master/examples"> Examples </a> | <a href="https://taichi.readthedocs.io/en/latest/contributor_guide.html"> Contributor Guidelines </a> | <a href="https://forum.taichi.graphics/"> Forum </a> </h3>
 </div>
 
-### Installation [![Downloads](https://pepy.tech/badge/taichi-nightly)](https://pepy.tech/project/taichi-nightly)
+## Overview
+
+Taichi (太极) is a high-performance programming language and compiler, designed for computer graphics applications. It has native support for spatially sparse computation and differentiable programming.
+
+<img src="https://github.com/yuanming-hu/public_files/raw/master/graphics/taichi/fractal_code.png" height="270px"> <img src="https://raw.githubusercontent.com/yuanming-hu/public_files/master/graphics/taichi/fractal.gif" height="270px">
```

Comment:
Is it possible to directly embed the code with this link..?

https://github.com/taichi-dev/taichi/blob/db8d2d6eb8c4b09a8405449086f4e9f5858e7f0f/examples/fractal.py#L1-L31

Reply:
Good idea. I just now tried that and it seems to take a huge space:

![Screenshot from 2020-04-14 19-54-51](https://user-images.githubusercontent.com/6553256/79284880-ea046f80-7e89-11ea-87bc-5836b26b654e.png)


How about adding a link to the code image to let it point to the actual file?

Reply:
Sure :)